zerver/lib/webhooks/common.py

```diff
@@ -114,3 +133,25 @@ def validate_extract_webhook_http_header(request: HttpRequest, header: str,
         raise MissingHTTPEventHeader(header)
 
     return extracted_header
+
+
+def get_http_headers(integration_name: str, fixture_name: str) -> Dict["str", "str"]:
+    """ This method is just a helper that finds and executes the correct fixture_to_headers method
+    for the given integration which would in turn generate the actual headers themselves. """
+
+    headers_module_name = "zerver.webhooks.{integration_name}.headers".format(
+        integration_name=integration_name)
+    try:
+        # Try to import the headers.py file. If it does not exist, then that means that the
+        # integration does not have any special http headers that are required.
+        headers_module = importlib.import_module(headers_module_name)
+    except ImportError:
+        return {}
+    try:
+        fixture_to_headers = headers_module.fixture_to_headers  # type: ignore # we do extra exception handling in case it does not exist below.
+    except AttributeError:
+        msg = " The {module_name} module must contain a fixture_to_headers method.".format(
+            module_name=headers_module_name
+        )
+        raise AttributeError(msg)
+    return(fixture_to_headers(fixture_name))
```

Comment:
I don't think the parenthesis are necessary here. `return fixture_to_headers(fixture_name)` should suffice here.